input/ipc-unix.c

```diff
@@ -302,7 +304,7 @@ static void *ipc_thread(void *p)
 
     struct mp_ipc_ctx *arg = p;
 
-    mpthread_set_name("ipc socket listener");
+    mpthread_set_name("ipc/listener/socket");
```

Comment:
This one just straight up exceeds the limit at 20 bytes.

Reply:
> This one just straight up exceeds the limit at 20 bytes.

It has literally the same length as before. Do you have suggestion, the limit is not a problem on normal operating systems.

Reply:
Maybe just go with `ipc/socket` for this one?

Reply:
`ipcc/` for client and `ipcl/` for listener could work, I guess. Only takes up 5 bytes.